src/main/java/echobot/Command.java

```diff
@@ -0,0 +1,367 @@
+package echobot;
+import echobot.task.*;
```

Comment:
Do not import using the * notation. Imported classes should always be listed explicitly.